openpype/hosts/nuke/plugins/load/load_model.py

```diff
@@ -15,13 +15,13 @@
 
 class AlembicModelLoader(load.LoaderPlugin):
     """
-    This will load alembic model into script.
+    This will load alembic model or anim into script.
     """
 
-    families = ["model"]
+    families = ["model", "pointcache", "animation"]
     representations = ["abc"]
 
-    label = "Load Alembic Model"
+    label = "Load Alembic Model or Anim"
```

Comment:
Might this be nicer if it's just more generic?
```suggestion
    label = "Load Alembic"
```

Reply:
Agreed :)